qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteCodeActions.java

```diff
@@ -100,14 +117,26 @@ public CompletableFuture<List<CodeAction>> doCodeActions(Template template, Code
 				QuteErrorCode errorCode = QuteErrorCode.getErrorCode(diagnostic.getCode());
 				if (errorCode != null) {
 					switch (errorCode) {
+					case UnknownProperty:

```

Comment:
You are using data to store part name but you can retrieve it easyly. You can castres the node in part if node is a part.